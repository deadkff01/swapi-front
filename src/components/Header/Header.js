import styled from 'styled-components'
import { Link } from '@reach/router'

const HeaderStyle = styled.div`
  font-family: 'STARWARS';
  color: #fff;
  font-size: 2rem;
  font-weight: 600;
  width: 100%;
  height: auto;
  top: 0;
  cursor: pointer;
  text-transform: uppercase;
  opacity: 0.9;
`

export default function Header() {
  return (
    <HeaderStyle className="w-full bg-black py-5 text-center mb-10">
      <Link to="/">Star Wars - Movies</Link>
    </HeaderStyle>
  )
}
