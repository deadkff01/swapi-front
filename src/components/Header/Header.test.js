import { cleanup } from '@testing-library/react';
import { renderWithRouter } from '../../testUtils';
import '../../setupTests';
import Header from './Header';

describe('Header component', () => {
  afterEach(() => {
    cleanup();
  });

  test('Renders the component', async () => {
    const { container } = renderWithRouter(<Header />);

    expect(container).toMatchSnapshot();
  });
});
