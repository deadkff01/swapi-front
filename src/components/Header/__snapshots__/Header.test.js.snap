// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Renders the component 1`] = `
<div>
  <div
    class="sc-bdfBwQ gCFjPG w-full bg-black py-5 text-center mb-10"
  >
    <a
      aria-current="page"
      href="/"
    >
      Star Wars Movies
    </a>
  </div>
</div>
`;
